src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs

```diff
@@ -277,6 +277,49 @@ public TimeSpan Expect100ContinueTimeout
             }
         }
 
+        public TimeSpan KeepAlivePingDelay
```

Comment:
I'm interested in what a documentation for these properties will be. What is the magical combination to turn the PING sending off?

Reply:
Yes, these properties require XML comments. Setting this to `Timeout.InfiniteTimeSpan` should turn it off.

Reply:
I've already written XML docs in ASP.NET Core. They might be a helpful starting point.

https://github.com/dotnet/aspnetcore/blob/29ceed2b2a6bcd9f3064f51cc78858e22c8544c4/src/Servers/Kestrel/Core/src/Http2Limits.cs#L149-L158

Reply:
Kestrel comments reused 👍